src/converter/FlameGraph.java

```diff
@@ -55,6 +57,8 @@ public FlameGraph(String... args) {
                 minwidth = Double.parseDouble(args[++i]);
             } else if (arg.equals("--skip")) {
                 skip = Integer.parseInt(args[++i]);
+            } else if (arg.equals("--highlightstacks")) {
+                highlightStacksPattern = Pattern.compile(".*" + args[++i] + ".*");
```

Comment:
Was on the fence whether the `.*` setup to allow single string matches should be pushed down to the argument or not